vortexasdk/utils.py

```diff
@@ -20,3 +23,29 @@ def filter_empty_values(data: Dict) -> Dict:
     return {
         k: v for k, v in data.items() if not (v is None or v == [] or v == {})
     }
+
+
+def get_latest_package_version(package_name: str):
+    """
+    Retrieves the latest SDK version from PyPI and compares with the currently installed SDK version.
+    """
+    url = f"https://pypi.python.org/pypi/{package_name}/json"
+    with urlopen(url) as u:
+        data = json.loads(u.read())
+
+    try:
+        versions = data["releases"].keys()
+    except KeyError:
+        raise KeyError("No version related key found in PyPI")
+
+    try:
+        versions = sorted(versions, key=StrictVersion)
```

Comment:
rather than mutate the `versions` variable here, we should create a new variable `sorted_versions`. We should generally write immutable variables when possible - it's easier to debug.